app-text/sigil/sigil-0.9.16.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit cmake-utils python-single-r1 readme.gentoo-r1 xdg-utils
+
+MY_P="Sigil-${PV}"
+
+DESCRIPTION="Sigil is a multi-platform WYSIWYG ebook editor for ePub format"
```

Comment:
No use of `${PN}` in `DESCRIPTION` please. I'd recommend starting with `Multi-platform ...`

Reply:
Done.